control/grpc.py

```diff
@@ -332,6 +370,12 @@ def __init__(self, config: GatewayConfig, gateway_state: GatewayStateHandler, rp
         self.max_hosts_per_subsystem = self.config.getint_with_default("gateway", "max_hosts_per_subsystem", GatewayService.MAX_HOSTS_PER_SUBSYS_DEFAULT)
         self.gateway_pool = self.config.get_with_default("ceph", "pool", "")
         self.ana_map = defaultdict(dict)
+        self.ana_grp_state = {}
+        self.ana_grp_ns_load = {}
+        self.ana_grp_subs_load = defaultdict(dict)
+        for i in range(MAX_POSSIBLE_ANA_GRPS):
+            self.ana_grp_ns_load[i] = 0
+            self.ana_grp_state[i] = 0xff #pb2.ana_state.INACCESSIBLE
```

Comment:
You can use pb2.ana_state.INACCESSIBLE. And, are you sure the value is 0xff and not 3?

Reply:
yes this  is ugly, will have to fix